tests/unit/helpers.js

```diff
@@ -78,7 +78,7 @@ export function getRuleFiles(ruleType) {
  */
 export function getVariable(scope, name) {
```

Comment:
question: @willdurand if I'm reading the diffs correctly, it looks like the getVariables function we use on the implementation side is now expecting to get the `variables` passed as the first parameter, while this similarly named one on the test side expected to get the `scope` object. I do wonder it we should consider keeping their type signature similar or rename one of the two to not be confused with each other (to be fair one is a test-only helper, I haven't dig into the historical reasons for having both of them around tbh).